EngineLayer/CrosslinkSearch/CrosslinkSpectralMatch.cs

```diff
@@ -34,7 +34,9 @@ public bool IsIntraCsm()
             
             if (this.ProteinAccession != null && this.BetaPeptide.ProteinAccession != null)
             {
-                if (this.ProteinAccession == this.BetaPeptide.ProteinAccession)
+                if (this.ProteinAccession == this.BetaPeptide.ProteinAccession || 
+                    this.ProteinAccession == "DECOY_"+ this.BetaPeptide.ProteinAccession || 
```

Comment:
does "this.ProteinAccession" really equal "DECOY_" and not something like "DECOY_P01234". Maybe this should just be looking for string to contain DECOY?

Reply:
It is "DECOY_ + BetaPeptide.ProteinAccession", which is treated as Intra-crosslink. Previously, this is treated as Inter-crosslink.